silverback/types.py

```diff
@@ -29,6 +34,31 @@ class SilverbackStartupState(BaseModel):
     last_block_processed: int
 
 
+class CronJob(BaseModel):
+    last_run: datetime = datetime.utcnow()
+    schedule: str
+
+    @property
+    def schedule_id(self) -> str:
+        return string_4byte(self.schedule)
+
+    @property
+    def task_name(self) -> str:
+        return f"cron/{self.schedule_id}"
+
+    def mark_ran(self):
+        self.last_run = datetime.utcnow()
+
+    def should_run(self) -> bool:
+        # NOTE: Checking against ONE_MINUTE since that is the resolution of cron schedules
+        return pycron.is_now(self.schedule) and datetime.utcnow() - self.last_run >= ONE_MINUTE
```

Comment:
`datetime.utcnow()` is deprecated, they suggest to call it with `datetime.now(datetime.UTC)`